### PR TITLE
Limits free shipping on goody orders in cargo

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
When I added goody crates in #51216, I had them ship in lockboxes to avoid paying the crate tax on small orders, and make orders easier to hand over. Unfortunately, my brokebrain failed to consider how much a pre-packed carryable cargo case can be abused

[](https://media.discordapp.net/attachments/326831214667235328/720671300950949968/unknown.png?width=380&height=300)

This PR clamps down on the above by limiting the free shipping cases to up to 6 orders at once. You can still order more things than the case can actual carry, but now you can only fit 6 eguns in there instead of infinite eguns. Above 6 orders, and you get a normal crate with the normal crate surcharge.

I'll probably have to add another var to supply orders to account for combat shotguns, which probably should not be able to be shipped 6 to a case, 

## Changelog
:cl: Ryll/Shaps
tweak: Nanotrasen's Central Shipping department has put limits on the free shipping it offers on goody crates, to discourage abuse of its bluespace packaging
/:cl:

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
does it make multiple boxes now?

**Comment by Ryll-Ryll** [comment]
No, tlal, the PR description says what it does

**Comment by Doosteroni** [comment]
LAME!! Everyone knows the classic 2 million eguns in the box trick like c’mon guys it’s obviously a feature

File: code/modules/shuttle/supply.dm [hunk 1]
```diff
@@ -100,6 +105,8 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 
 	var/value = 0
 	var/purchases = 0
+	var/list/goodies_by_buyer = list() // if someone orders more than GOODY_FREE_SHIPPING_MAX goodies, we upcharge to a normal crate so they can't carry around 20 combat shotties
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/shuttle/supply.dm:108]: :waaa_fb:

Comment (Ryll-Ryll) [reply code/modules/shuttle/supply.dm:108]: what does this mean

Comment (ExcessiveUseOfCobblestone) [reply code/modules/shuttle/supply.dm:108]: whoops i thought this was under a class

**Pull request merged**